jena-text/src/test/java/org/apache/jena/query/text/assembler/TestGenericAnalyzerAssembler.java

```diff
@@ -0,0 +1,198 @@
+package org.apache.jena.query.text.assembler;
```

Comment:
No problem - It is not an issue specific to this class. I noticed it and found it was common in the text tests.